assets/js/index.js

```diff
@@ -7,6 +8,7 @@ require('bootstrap');
 require('select2');
```

Comment:
@nicolaisueper `'select2'` should be removed. This is included because @revrng wanted to test this. I’ll open another ticket for that.